README.md

```diff
@@ -150,6 +152,64 @@ Some things to note about utility classes:
 
 7. Components and utilities are found in tests/visual/ directory.
 
+#### Working process
+
+The process below describes how to create a new component. The steps are described using `components` but the process is identical for `utils`. When building a new utility, simply replace all references to `components` with `utils`. For example: instead of `/dist/components/component-name/component-name.scss`, it's `/dist/utils/utils-name/utils-name.scss`.
+
+1. Create a new component folder in **/dist/component/{name-of-your-component}**
+
+2. In your new component folder create the following files:
+
+    _(For a good starting place, see the template files in **/templates**)_
+
+    ```
+    /dist/components/component-name
+    -- _component-name.scss
+    -- component-name.dust
+    -- component-name.json
+    -- readme.md
+    ```
+    Note that the `*.scss` file should be prefixed with an underscore to avoid creating an unnecessary css file.
+
+3. Create another component folder, but this time in **/tests/visual/components**
```

Comment:
Steps 3 through 7 are about writing tests. Only steps 1 and 2 are about building the component. It would be clearer if these two processes were better delineated. Either it should be two separate lists, or  we need something in the text that says like: “Next, we should write some visual tests for our component. Create a new folder in `tests`… etc.

Also, step seven is a summary, not an actual step, and should be included as part of one of the existing steps.